lib/ancestry/instance_methods.rb

```diff
@@ -139,7 +139,11 @@ def ancestors?
     alias :has_parent? :ancestors?
 
     def ancestry_changed?
-      changed.include?(self.ancestry_base_class.ancestry_column.to_s)
+      if ActiveRecord::VERSION::STRING >= '5.1.0'
+        will_save_change_to_attribute?(self.ancestry_base_class.ancestry_column.to_s)
```

Comment:
@jrafanie thanks

looks like this in 2017, `will_save_change_to_attribute` changes again.
In 5.1, does `saved_change_to_attribute?` also meet your needs?
Or should we put a double if there to support the newer versions

Reply:
I'll have to take a look. Thanks

Reply:
Ok, I took a look further.  `saved_changed_to_attribute?` is for after save callbacks to indicate we just saved changes to a specific attribute.  Before save callbacks much ust the will_save...

see https://www.ombulabs.com/blog/rails/upgrades/active-record-5-1-api-changes.html

Reply:
oh wait, it looks like this method is called from both before and after save callbacks.  I'll have to fix this.

Reply:
I need to find a cleaner way of implementing this.  It's fixed in two different places for "before save" using "will save changes" and "after save" "saved changes to".